prismic/custom-types/badger.json

```diff
@@ -60,16 +60,38 @@
         "label" : "Github"
       }
     },
+    "primary-image" : {
+      "type" : "Image",
+      "config" : {
+        "constraint" : {
+          "width" : 670,
+          "height" : 670
+        },
+        "thumbnails" : [],
+        "label" : "Primary image"
+      }
+    },
+    "secondary-image" : {
+      "type" : "Image",
+      "config" : {
+        "constraint" : {
+          "width" : 670,
+          "height" : 670
+        },
+        "thumbnails" : [],
+        "label" : "Secondary image"
+      }
+    },
     "image-url" : {
       "type" : "Text",
       "config" : {
-        "label" : "Image URL – see instructions here: http://webdocs.red-badger.com/chapter1.html"
+        "label" : "DEPRECATED - use primary-image"
```

Comment:
staging JSON doesn't matter you can edit it on the fly. What's matter is the production JSON. Label might be different if they weren't updated in staging via uploading a JSON file